python/ray/tests/test_runtime_context.py

```diff
@@ -346,6 +346,19 @@ def foo(self):
     ray.get(actor.foo.remote())
 
 
+def test_actor_name(ray_start_regular):
```

Comment:
nit can you just add this test in the previous test test_ids? Having a new suite will do ray.init which is pretty slow (and I think this test is too small, so it is probably better just piggybacking to the previous test)

Reply:
Gotcha!